FLIR/conservator/cli/managers.py

```diff
@@ -165,6 +165,11 @@ def download(
             is_flag=True,
             help="Include child directories recursively, creating collections as needed",
         )
+        @click.option(
+            "--resume-media",
+            is_flag=True,
+            help="Check if media files were previously uploaded, skip if so",
```

Comment:
Do you think `--resume-media` should be the default behavior?

Reply:
The overhead is an extra phase at the beginning of each folder querying for all the media in it. If you've got 1200 videos like in some projects that could be substantial. I figured somebody should know if they tried to upload a directory before or not, so they would know when to pass in resume.

That said, I wouldn't be averse to flipping the default. Thoughts?

Reply:
That's a good point, and I guess there are tradeoffs both ways. 

The current implementation is on a per-local-file basis, checking if collection has a match of the single name. We *could* fetch all media at once on a per-collection basis and compare names ourselves. Would likely be a significant speedup, would have minimal slowdown when uploading to an empty collection. 

I guess I'd slightly prefer switching to resume by default and optimizing, but am fine if you want to keep it as-is.